src/main/java/com/github/hansi132/discordfab/discordbot/integration/UserSynchronizer.java

```diff
@@ -125,17 +124,31 @@ public static void sync(@NotNull final PrivateChannel channel, @NotNull final Us
                 updateStatement.execute();
 
                 OnlineUser onlineUser = KiloServer.getServer().getOnlineUser(UUID.fromString(mcUUID));
```

Comment:
Instead of getting an OnlineUser try to get a User.
Try something like 
```
KiloEssentials.getUserThenAcceptAsync(UUID.fromString(mcUUID), (user) -> { 
    //Code Here 
})
```

Reply:
This is not changed in the original code, so i wont change it in this pr, I think it was drex that implemented that the first time, could ask him if we should do it or not

Reply:
We should. if the Player isn't online then it'll return `null` which doesn't seem like a very good thing.